ghost/core/core/server/services/members/service.js

```diff
@@ -74,6 +74,36 @@ const processImport = async (options) => {
     return result;
 };
 
+const updateVerificationTrigger = () => {
```

Comment:
This changes was required to make testing easier because  the `VerificationTrigger` is not updated after config changes.